src/js/components/conversation.jsx

```diff
@@ -94,21 +95,24 @@ export class ConversationComponent extends Component {
     };
 
     scrollToBottom = () => {
-        const {shouldScrollToBottom} = this.props;
-        if(!this._isScrolling && (shouldScrollToBottom || this._forceScrollToBottom)) {
+        const {shouldScrollToBottom, quickReplies} = this.props;
+        if (!this._isScrolling && (shouldScrollToBottom || this._forceScrollToBottom)) {
             this._isScrolling = true;
-            const timeout = setTimeout(() => {
```

Comment:
I feel like we keep going back on forth on whether or not `setTimeout` is required. What prompted the change this time?